Move fx2trt out of PyTorch core (#740)

yinghai opened the pull request:
Summary:
Pull Request resolved: https://github.com/pytorch/benchmark/pull/740

To fx2trt out of tree to remove bloatness of PyTorch core.

It's the first and major step. Next, we will move acc_tracer out of the tree and rearrange some fx passes.

Differential Revision: D34065866

suo commented on test/fx2trt/core/test_import_fx2trt.py:
```diff
@@ -5,7 +5,7 @@
 # Test that this import should not trigger any error when run
 # in non-GPU hosts, or in any build mode.
-import torch.fx.experimental.fx2trt.lower as fxl  # noqa: F401
+import fx2trt_oss.fx.lower as fxl  # noqa: F401
```
For my understanding—is the stuff in `test/fx2trt` eventually supposed to go as well?

yinghai replied:
Yes. They will need to go. I'm actually surprised that they are here.

suo commented:
One small note, @wushirong added tensorrt to the PyTorch OSS CI docker images I believe. Are those removable after fx2trt moves out of core?

suo commented on test/fx2trt/trt_lower/test_diagnostics.py:
```diff
@@ -0,0 +1,184 @@
+# Owner(s): ["oncall: aiacc"]
```
is this file supposed to be added?

yinghai replied:
Sorry this is the file that this diff was rebased on. Should be gone already.

Commit:
Move fx2trt out of PyTorch core (#72499)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/72499

Pull Request resolved: https://github.com/pytorch/benchmark/pull/740

To fx2trt out of tree to remove bloatness of PyTorch core.

It's the first and major step. Next, we will move acc_tracer out of the tree and rearrange some fx passes.

Reviewed By: suo

Differential Revision: D34065866

fbshipit-source-id: 478dfd491b88cc14a677d66d82342c99e4e468d5

yinghai commented:
> One small note, @wushirong added tensorrt to the PyTorch OSS CI docker images I believe. Are those removable after fx2trt moves out of core?

Yeah. I'm asking @wushirong to remove those.
